expat/coverage.sh

```diff
@@ -41,6 +41,10 @@ _configure() {
     ${unicode_enabled} \
             && configure_args+=( CPPFLAGS='-DXML_UNICODE -DXML_UNICODE_WCHAR_T' )
 
+    if [ "${xml_attr_info}" = true ]; then
```

Comment:
Please use Bash `[[` (double square brackets) as the rest of `coverage.sh`. Thanks!

Reply:
Addressed review comments in latest patchset